showcase/app/styles/showcase-pages/form.scss

```
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// FORM LAYOUT
+
+@use "../typography" as *;
+
+body.components-form {
```

[suggestion] more specific (otherwise the selector would apply also to the other form pages)
```suggestion
body.components-form-layout {
```